src/module.c

```diff
@@ -71,7 +71,13 @@ static uintptr_t dc_event_handler(dc_context_t* dc_context, int event, uintptr_t
   }
 
   // Start tracing events here. DC_EVENT_GET_STRING is just too spammy.
-  TRACE("event %d", event);
+#ifdef DEBUG
+  if (event == DC_EVENT_INFO) {
+    TRACE("event INFO %s", (char*)data2);
```

Comment:
We get some INFO events when closing the context and might want to print them out during DEBUG.